src/popover/container.tsx

```diff
@@ -57,9 +59,10 @@ export default function PopoverContainer({
   const arrowRef = useRef<HTMLDivElement | null>(null);
   const previousInternalPositionRef = useRef<InternalPosition | null>(null);
 
-  const [popoverStyle, setPopoverStyle] = useState<CSSProperties>(INITIAL_STYLES);
+  const [popoverStyle, setPopoverStyle] = useState<Offset>(INITIAL_STYLES);
   const [internalPosition, setInternalPosition] = useState<InternalPosition | null>(null);
   const isRefresh = useVisualRefresh();
+  const isMobile = useMobile();
```

Comment:
`isMobile` is just a constant value of some width. Why should 700px and 600px wide screens work differently?